rooms/views.py

```diff
@@ -8,3 +8,9 @@ class RoomsList(ListView):
     """Lists all the rooms in the school that can be reserved by students and staff"""
     model = Room
     template_name = 'rooms/rooms_list.html'
+
+
+class RoomDetails(DetailView):
+    """"""
+    model = Room
+    template_name = 'rooms/room_details.html'
```

Comment:
NICE! Go ahead and add a description in the docstring though (the `"""` bit)

Reply:
Done